lib/Demangling/Demangler.cpp

```diff
@@ -2160,6 +2160,7 @@ NodePointer Demangler::demangleImplParamConvention(Node::Kind ConvKind) {
     case 'l': attr = "@inout"; break;
     case 'b': attr = "@inout_aliasable"; break;
     case 'n': attr = "@in_guaranteed"; break;
+    case 'C': attr = "@in_cxx"; break;
```

Comment:
This should be documented in docs/ABI/Mangling.rst.

Reply:
Also, please add a test in https://github.com/apple/swift/blob/main/test/Demangle/Inputs/manglings.txt